F1WM/Controllers/NewsController.cs

```diff
@@ -49,7 +49,100 @@ public async Task<IActionResult> GetSingle(int id)
 			}
 		}
 
-		[HttpGet("important")]
+        [HttpGet("tag/{id}")]
+        [Produces("application/json", Type = typeof(NewsSummary))]
```

Comment:
It produces IEnumerable<NewsSummary>, but anyway I'd like you to change the return type of the method. See other comments.